core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1889,6 +1889,14 @@ package object config {
       .doubleConf
       .createWithDefault(0.75)
 
+  private[spark] val SPECULATION_MIN_THRESHOLD =
+    ConfigBuilder("spark.speculation.min.threshold")
```

Comment:
According to the [config naming policy](https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/internal/config/ConfigEntry.scala#L21), `spark.speculation.shortTaskThreshold` may be better?

cc @Ngone51

Reply:
I'm fine with renaming, though I don't care for shortTaskThreshold.  why not just minThreshold or minTaskThreshold

Reply:
`minTaskRuntime`? I admit that I'm not good at naming ...

Reply:
that works for me